suite_cam6.xml

```diff
@@ -0,0 +1,361 @@
+      <scheme>zm_convr</scheme>                 <!-- Actual deep convection routine -->
+      <scheme>zm_convr_cam_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>zm_conv_evap</scheme>             <!-- Actual ZM rain evaporation scheme -->
+      <scheme>zm_conv_evap_cam_update</scheme>  <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>momtran</scheme>                  <!-- Actual ZM momentum transport scheme -->
+      <scheme>momtran_cam_update</scheme>       <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>convtran</scheme>                 <!-- Actual ZM convective transport scheme, for constituents in cnst_is_convtran1 -->
+      <scheme>convtran_cam_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <!-- **************************** -->
+      <scheme>check_energy_chng</scheme>        <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_tracers_chng</scheme>       <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <scheme>sslt_rebin_adv</scheme>           <!-- Only matters if certain sea salt species are present in chemistry package-->
+      <subcycle loop="cloud_macmic_num_steps">  <!-- Cloud macrophysics-microphysics loop -->
+        <!-- CLUBB turbulence, convection, and macrophysics -->
+        <!-- **************************** -->
+        <!--This section will likely require additional clean-up,
+            as many of these routines are simply unit or vertical
+            grid conversions, or the re-naming and re-setting
+            of host model inputs.  Thus they can likely be
+            managed by the framework itself, and won't need
+            their own scheme calls.-->
+        <scheme>set_dry_to_wet</scheme>               <!-- Convert mixing ratios from dry air to air + water vapor -->
+        <scheme>grid_size</scheme>                    <!-- Calculate grid-cell size (not sure if this will be managed by registry/interface?) -->
+        <scheme>clubb_dtime_calc</scheme>             <!-- Calculate CLUBB time-step as function of host model time-step -->
+        <scheme>exner_clubb</scheme>                  <!-- Calculate Exner function using CLUBB definition (should be manageable via the registry/meta file) -->
+        <scheme>tropopause_findChemTrop</scheme>      <!-- Calculate tropopause height -->
+        <scheme>clubb_vertical_grid_inv</scheme>      <!-- Change order of vertical levels (should be manageable by registry/meta file) -->
+        <scheme>clubb_calc_ustar</scheme>             <!-- Only matters for single-column mode -->
+        <scheme>setup_grid_heights_api</scheme>       <!-- Re-calculate CLUBB grid heights -->
+        <scheme>setup_parameters_api</scheme>         <!-- Re-calculate grid-dependent CLUBB parameters -->
```

Comment:
This might be worth discussing, as I could imagine both pros and cons.  On the one hand I can see the argument that the suite definition file should include all the separate CAM schemes (without additional interface layers), as that way a user (either a scientist or engineer) can quickly see exactly what CAM is doing, and in what order.  

On the other hand, the CLUBB interface  is quite tricky because several sections of calculations depend on namelist variables (like `do_rainturb`), and so if we included every one of those options as "execute" statements it could possibly become difficult to read the suite definition file.   I guess we should wait and see if @gold2718  has any strong opinions on the matter.